src/map/time_server.cpp

```diff
@@ -100,6 +100,7 @@ int32 time_server(time_point tick,CTaskMgr::CTask* PTask)
 				});
 			});
 
+            guildutils::UpdateGuildPointsPattern();
```

Comment:
Moving this call to here will call the update function every vanadiel day. It checks `CVanaTime::getInstance()->getSysYearDay()` so it's still going to be using the server timezone. It also refreshes all the guilds patterns, even if it didn't change, so creating a lot of work for no real end result. I think to get the result you want you would need to do something like adding `luautils::GetSettingsVariable("TIMEZONE_OFFSET")` to the midnight check, but then you'd need to also deal with `guildutils::UpdateGuildPointPattern()` since it uses `CVanaTime::getInstance()->getSysYearDay()` as the server var.